spec/features/user_can_edit_horse_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../spec_helper'
+
+describe "When user edits a horse and clicks submit" do
+  it "changes the appropriate attribute for that horse" do
+    Breed.create(name: "Palimino")
+    Jockey.create(name: "Adolf")
+    Horse.create(name: "Phil", age: 22, total_winnings: 3, breed_id: 1, jockey_id: 1)
+
+    visit('/horses/1/edit')
+
+    horse = Horse.find(1)
+    expect(horse.name).to eq("Phil")
+
+    fill_in('horse[name]', :with => 'Dufus')
+    click_button('Update Horse')
+
+    horse = Horse.find(1)
+    expect(horse.name).to eq("Dufus")
```

Comment:
Similar comment here regarding user interaction. One additional note: if you save the horse you create in line 7 to a variable you can reference that variable later in the test. In order to check to see that his name has changed, you'll need to run `horse.reload` to get the updated information from the database.